test/runtests.jl

```diff
@@ -5,6 +5,11 @@ using AbstractFFTs: Plan, plan_inv
 using Test
 using LinearAlgebra
 
+using Aqua
+@testset "Project quality" begin
+    Aqua.test_all(FFTW, stale_deps=false, piracies=false)
```

Comment:
```suggestion
    Aqua.test_all(FFTW; stale_deps=FFTW.fftw_provider=="fftw", piracies=false)
```
no?

Reply:
Don't think this will work, as `MKL_jll` won't be loaded if the provider is `fftw`, and similarly `FFTW_jll` won't be loaded if the provider is `mkl`.

Reply:
Ah, right, I missed the first version was `stale_deps=fftw_provider=="mkl"`